tests/desktop/test_desktop_api_browse.py

```
@@ -0,0 +1,120 @@
+
+        with patch.dict("sys.modules", {"webview": mock_webview}):
+            api = DesktopAPI()
+            result = api.browse_directory()
+
+        assert result == "/Users/rahul/Documents"
+
+    def test_calls_folder_dialog_constant(self) -> None:
+        """Must call create_file_dialog with FOLDER_DIALOG, not a file dialog."""
+        from file_organizer.desktop.app import DesktopAPI
+
+        mock_webview, mock_window = self._make_mock_webview(("/tmp/test",))
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/test"

(S108)